Content.Client/GameTicking/Managers/ClientGameTicker.cs

```diff
@@ -43,6 +43,8 @@ public sealed class ClientGameTicker : SharedGameTicker
 
         public override void Initialize()
         {
+            base.Initialize();
```

Comment:
Looking at the code, this is gonna make client-side replays have invalid 0 round IDs. Should probably guard against that?

Reply:
`replay.yml` has `roundId: 0` instead of not having it at all. Seems harmless to me, but I added a `!= 0` guard before writing it to be on the safe side.